packages/react/src/components/form/BaseInput/index.tsx

```diff
@@ -1,18 +1,18 @@
 import styled, { css } from "styled-components";
 import { typography, TypographyProps } from "styled-system";
-import React, { InputHTMLAttributes } from "react";
+import React, { InputHTMLAttributes, useCallback } from "react";
 import FlexBox from "../../layout/Flex";
 import Text from "../../asorted/Text";
 import { rgba } from "../../../styles/helpers";
 
-export type CommonProps = InputHTMLAttributes<HTMLInputElement> &
+export type CommonProps = Omit<InputHTMLAttributes<HTMLInputElement>, "onChange"> &
   TypographyProps & {
     disabled?: boolean;
     error?: string;
   };
 
 export type InputProps = CommonProps & {
-  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  onChange: (e: string) => void;
```

Comment:
⚠️ This is a breaking change, so we'll need to ensure that the work done in LLD so far gets updated if necessary.

Reply:
i dont think we used or will use any input just yet so it should be fine for now :+1: